widgets/contact/contact.php

```diff
@@ -913,8 +913,8 @@ function render_form_label( $field_id, $label, $position ) {
 	 * Ajax action handler to send the form
 	 */
 	function contact_form_action( $instance, $storage_hash ) {
-		if ( ! wp_verify_nonce( $_POST['_wpnonce'] ) ) {
-			exit();
+		if ( ! wp_verify_nonce( $_POST['_wpnonce'], '_contact_form_submit' ) ) {
+			return false;
```

Comment:
Yeah, this is where the issue occurred with Elementor. The `contact_form_action` function is called from the widget template file, which is sometimes rendered as a means to enqueue scripts required by the front end, e.g. for live previews. In those cases `$_POST['_wpnonce']` doesn't exist and then `wp_die` halts all script execution and breaks things. I'll add a comment there.